packages/store-ui/src/atoms/Bullets/Bullets.tsx

```diff
@@ -0,0 +1,44 @@
+import type { MouseEvent } from 'react'
+import React from 'react'
+
+import Button from '../Button'
+
+export interface BulletsProps {
+  totalQuantity: number
+  activeBullet: number
+  onClick: (e: MouseEvent, bulletIdx: number) => void
+  testId?: string
+}
+
+function Bullets({
+  totalQuantity,
+  activeBullet,
+  onClick,
+  testId = 'store-bullets',
+}: BulletsProps) {
+  const bulletIndexes = [...new Array(totalQuantity).keys()]
+
+  return (
+    <ol data-store-bullets data-testid={testId}>
+      {bulletIndexes.map((idx) => {
+        const isActive = activeBullet === idx
+
+        return (
+          <li
+            key={idx}
+            data-testid="bullet-item"
+            data-bullet-item
+            data-active={isActive || undefined}
+          >
+            <Button
+              aria-label={isActive ? 'Current page' : `Go to page ${idx + 1}`}
```

Comment:
this should be internationalizable at some point, right?

Reply:
Yes! It should!
I did a quick research on this and apparently, browsers are moving towards automatic translating `aria` attributes (Chromium-based browsers already do it, and Safari too), but there are still some caveats.

Do we already have a good solution for i18n inside of store-ui components? Or should I just turn this into a prop and expect the user to handle it?

Reply:
The ideia is not to have an internationalization tool inside store-ui, but rather let the user handle it